src/NzbDrone.Core/NetImport/NetImportSearchService.cs

```diff
@@ -81,19 +81,7 @@ public List<Movie> MovieListSearch(int listId, bool onlyEnableAuto = false)
 
             _logger.Debug("Found {0} movies from list(s) {1}", movies.Count, string.Join(", ", lists.Select(l => l.Definition.Name)));
 
-			return movies.DistinctBy(x => {
-				if (x.TmdbId != 0)
-				{
-					return x.TmdbId.ToString();
-				}
-
-				if (x.ImdbId.IsNotNullOrWhiteSpace())
-				{
-					return x.ImdbId;
-				}
-
-				return x.Title;
-			}).ToList();
```

Comment:
Totally sorry for the merge f'ups but looked for the first time of my life at a kdiff3 or some such with 3 panels. Think I kinda figured it out. "Theirs" is the branch you are merging from, "Ours" the branch we are merging into and the middle screen is basically the result.